cryptomator/src/main/java/ch/cyberduck/core/cryptomator/CryptoVault.java

```diff
@@ -223,10 +223,10 @@ public synchronized CryptoVault load(final Session<?> session, final PasswordCal
         return this.unlock(session, prompt, bookmark, passphrase);
     }
 
-    protected VaultConfig readVaultConfig(final Session<?> session) throws BackgroundException {
+    protected VaultConfig getVaultConfigFromJWT(final Session<?> session) throws BackgroundException {
         try {
-            final String token = new ContentReader(session).read(config);
-            return this.readVaultConfig(token).withMasterkeyFile(this.readMasterkeyFile(session, masterkey));
+            final String token = readVaultConfig(session);
+            return this.getVaultConfigFromJWT(token).withMasterkeyFile(this.readMasterkeyFile(session, masterkey));
```

Comment:
Redundant to read master key for new vault in `create`

Reply:
@dkocher  `readVaultConfigFromJWT` is called in `unlock` - could you expand on your comment?

Reply:
Was mistaking method as they are named the same with different both static and instance variants with different parameters.